libc/utils/hdrgen/yaml_to_classes.py

```diff
@@ -244,7 +241,10 @@ def main():
         help="Add a function to the YAML file",
     )
     parser.add_argument(
-        "--e", action="append", help="Entry point to include", dest="entry_points"
+        "--entry-point",
```

Comment:
Is it worth keeping a short form of `-e` as you did for main.py? E2BIG command lines come to mind.

Reply:
I consider all this command-line support to be vestigial.  My intent was just to avoid breaking existing uses until they are all migrated to revamped command-line interfaces yet to be decided upon.